ci/docker/python-wheel-musllinux-test.dockerfile

```
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG alpine_linux
+ARG python_image_tag
+FROM python:${python_image_tag}-alpine${alpine_linux}
+
+RUN apk add --no-cache python3-dev g++ bash linux-headers tzdata
```

Could you sort package list?

```suggestion
RUN apk add --no-cache bash g++ linux-headers python3-dev tzdata
```